src/convertToObject.js

```diff
@@ -15,9 +15,9 @@ function convertToObject(sourceString) {
   const arr = sourceString.split(';\n').filter(elem => elem.includes(':'));
   const obj = {};
 
-  for (const elem of arr) {
+  arr.forEach(function(elem) {
```

Comment:
```suggestion
  arr.forEach((elem) => {
```
use arrow functions in methods